AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -249,7 +268,7 @@ - (void)startUpdate {
       return;
     }
     NSString *distributionGroupId = [MS_USER_DEFAULTS objectForKey:kMSDistributionGroupIdKey];
-    if (updateToken || distributionGroupId) {
+    if (updateToken || distributionGroupId || MSDistribute.updateTrack == MSUpdateTrackPublic) {
```

Comment:
Remove `|| distributionGroupId` from this check.
It was there because we were getting the public group via the browser redirection without token but this is no longer the case with public. We need only to test if we have token or if current track is public.